src/lib.rs

```diff
@@ -751,6 +844,36 @@ pub unsafe trait FromZeroes {
     /// Self::new_zeroed()`, it differs in that `zero` does not semantically
     /// drop the current value and replace it with a new one - it simply
     /// modifies the bytes of the existing value.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # mod derive { pub use zerocopy_derive::FromZeroes; }
+    /// # use zerocopy::FromZeroes;
+    /// # use derive::*;
```

Comment:
```suggestion
    /// # use zerocopy::FromZeroes;
    /// # use zerocopy_derive::*;
```

Same for other examples.

Reply:
Also filed an issue based on this: https://github.com/google/zerocopy/issues/539